src/dynamic_compiler.c

```diff
@@ -2432,6 +2479,9 @@ char *dynamic_compile_split(char *ct) {
 int dynamic_assign_script_to_format(DC_HANDLE H, struct fmt_main *pFmt) {
 	int i;
 
+	/* assume comipler success. */
```

Comment:
Small typo here.

Reply:
numerous small errors (mostly signed/unsigned char*), but this one also.  All fixed, and are in 2nd push.
The new format should handle any failing expressions.  It has no optimizations, and is slow due to marshal code at each step, but it is fast enough to get the job done, AND if the expression IS a common expression (i.e. used a lot), then we can see about working them into a real format, whether it be with a dynamic script, or a fat format.

Reply:
Will be fixed next time I check things in.

Reply:
Fixed in 39ad0fe